taqueria-plugin-archetype/README.md

```diff
@@ -1,13 +1,12 @@
 # Archetype Plugin
 
-This is a plugin developed for Taqueria built on NodeJS using the Taqueria Node SDK and distributed via NPM
-
 The Archetype plugin provides a task to compile Archetype smart contracts (`.arl`) to Michelson `.tz` files
 
 ## Requirements
 
-- Node.js v16.3 or later
-- Docker v0.8.6 or later
+- Taqueria v0.20.2 or later
+- Node.js v16.17.1 or later
+- Docker v20.10.12 or later
```

Comment:
Essentially, we decided to post the versions that we test with, to avoid surprises.
An option is to have test machines that test with an older setup.

Reply:
@mweichert I put in a recent version that looked not too far off from what we are all using. I wanted one released within the last year. If we want this to be a definitive "minimal version" I think we should put in something to point people to the fact that they can also use a recommended version